tgbridge/cmd/server/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"log"
+	"tg-bridge/tgbridge/internal/config"
+	"tg-bridge/tgbridge/internal/tgsession"
+
+	"github.com/gotd/td/session"
+	"github.com/gotd/td/telegram"
+)
+
+func main() {
+	cfg := config.LoadConfig()
+
+	sessionStorage, err := tgsession.CreateSessionStorage(cfg)
+	client := CreateTelegramClient(cfg, sessionStorage)
+
+	err = client.Run(context.Background(), func(ctx context.Context) error {
```

Comment:
let's consider you will have here in future more background taks - at least it will be required also prometheus metrics been provided and ready/health endpoint, so it would be good one to wrap it into goroutine.